app/models/collection_statistic.rb

```diff
@@ -1,45 +1,45 @@
 module CollectionStatistic
   def work_count
-    self.works.count
+    works.count
   end
 
   def page_count
-    Collection.count_by_sql("SELECT COUNT(*) FROM pages p INNER JOIN works w ON p.work_id = w.id WHERE w.collection_id = #{self.id}")
+    Page.count_by_sql("SELECT COUNT(*) FROM pages p INNER JOIN works w ON p.work_id = w.id WHERE w.collection_id = #{self.id}")
   end
 
-  def subject_count(last_days=nil)
-    self.articles.where("#{timeframe_clause(last_days, 'created_on')}").count
+  def subject_count(last_days=nil, *)
+    self.articles.where("#{last_days_ago_clause(last_days, 'created_on')}").count
```

Comment:
I added the `, *` argument to several of these methods here in the `collections_statistic.rb` so that it could share the `UserMailer::StastisticWrapup` builder with the `owner_statistic.rb`. Changing the argument structure felt risky to me, so I tested out different scenarios in the console to make sure it didn't break things. Plus all of the tests pass. But I'd love talk through this change as well.

Reply:
I'd never heard of the [`, *` argument system](https://github.com/ruby/spec/blob/master/language/def_spec.rb#L83)!  Now I've learned something today.

Your change makes sense to me.